tests/unit/test_polynomialembeddor.py

```diff
@@ -0,0 +1,343 @@
+# Copyright 2016 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# ================================================================================================
+
+import unittest
+
+from collections import defaultdict
+
+from dwave.embedding.polynomialembeddor import processor, _bulk_to_linear, _to_linear, _bulk_to_chimera
+from dwave.embedding.polynomialembeddor import _chimera_neighbors, random_processor, _to_chimera
+
+
+def verify_chains(Proc, emb):
+    proc = Proc._proc0
```

Comment:
What's the thinking behind the `_proc0` attribute? Asking because I don't  any references to `_proc1` or higher.